### PR TITLE
Extend online key support to SaaS

### PR DESCRIPTION
**Pull request by kairoaraujo**
# Description

Adds support to use Online Key from many different SaaS/Containers

- Azure KMS
- AWS KMS
- Google Cloud KMS
- HashiCorp Key Vault

Fixes #https://github.com/repository-service-tuf/repository-service-tuf/issues/343 
Fixes #https://github.com/repository-service-tuf/repository-service-tuf/issues/24

# Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality not to work as expected)

# Additional requirements
- [x] Tests have been added for the bug fix or new feature
- [ ] Docs have been added for the bug fix or new feature

# Code of Conduct
By submitting this PR, you agree to follow our [Code of Conduct](CODE_OF_CONDUCT.rst).

- [x] I agree to follow this project's Code of Conduct

### CODE REVIEW

File: repository_service_tuf/cli/admin/helpers.py [hunk 1]
```diff
@@ -261,6 +275,47 @@ def _load_key_prompt(
     return key
 
 
+def _load_key_online_prompt(
```
Comment (MVrachev) [inline repository_service_tuf/cli/admin/helpers.py:278]:
```suggestion
def _load_online_key_prompt(
```

**Review by MVrachev — COMMENTED** [review]
Great job bringing many awesome features!
It's so awesome we can just reuse securesystemslib work in such a way!

I left one small nit.

File: repository_service_tuf/cli/admin/helpers.py (hunk 1 above)
Comment (kairoaraujo) [reply repository_service_tuf/cli/admin/helpers.py]: Added in the fixup

**Comment by kairoaraujo** [comment]
@MVrachev

Please merge first https://github.com/repository-service-tuf/repository-service-tuf/pull/803 when you think the PR is good, so when approving the FT will be running.

**Review by MVrachev — APPROVED** [review]

**Review by MVrachev — APPROVED** [review]

**Review by MVrachev — APPROVED** [review]

**Review by MVrachev — APPROVED** [review]

**Review by MVrachev — APPROVED** [review]

**Pull request merged**